assets/src/edit-story/masks/index.js

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import StoryPropTypes from '../types';
+
+// Important! This file cannot use `styled-components` or any stateful/context
```

Comment:
@swissspidy can you help here? Can we construct a eslint rule to prohibit most of imports here. Specifically we need to disallow `styled-components` anywhere in this folder.

Reply:
You can add an `.eslintrc` file in this folder with the following contents:

```json
{
  "rules": {
    "no-restricted-imports": ["error", { "paths": ["styled-components"] }]
  }
}
```

Can amend the list with `@wordpress/element` (React) and others if needed.

Reply:
Awesome! Thanks!